test/org/sagebionetworks/bridge/dynamodb/DynamoCriteriaDaoTest.java

```diff
@@ -0,0 +1,79 @@
+package org.sagebionetworks.bridge.dynamodb;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+import javax.annotation.Resource;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import org.sagebionetworks.bridge.models.Criteria;
+
+import com.google.common.collect.Sets;
+
+@ContextConfiguration("classpath:test-context.xml")
+@RunWith(SpringJUnit4ClassRunner.class)
+public class DynamoCriteriaDaoTest {
+
+    @Resource
+    DynamoCriteriaDao criteriaDao;
+
+    @Test
+    public void deleteAndGetAreQuiet() {
+        // Before creating an object, create and delete do not throw errors and do something sensible
+        criteriaDao.deleteCriteria("key"); // no exception
+        Criteria retrieved = criteriaDao.getCriteria("key"); // just return null
+        assertNull(retrieved);
+    }
```

Comment:
If for any reason, multiple unit tests are running at the same time, there's a chance that this might collide and not do what you expect it to. Would it be better to use a random key?